frontend/src/locales/en.json

```
@@ -317,6 +358,7 @@
       "continueToFxa": "Enter your email above to continue to Mozilla Accounts"
     },
     "nameIsInvitingYou": "{name} is inviting you",
+    "ownerNeedsToConfirmBooking": "If this option is active, appointments are only booked until the calendar owner confirmed it.",
```

```suggestion
    "ownerNeedsToConfirmBooking": "When this option is active, you will be required to confirm or decline times booked on your schedule. When this option is deactivated, all times will automatically be confirmed.",
```
I almost think this should show two information text depending on the state of the button. But I suggested both here.

---

Either way it's their calendar, so we should be referring to them not the calendar owner.

---

Yes thanks. I added your suggestions. If we feel this is too much text, we can conditionally show parts of it.